docs/modules/pkl-cli/partials/cli-common-options.adoc

```diff
@@ -19,6 +19,19 @@ Patterns are matched against the beginning of resource URIs.
 At least one pattern needs to match for a resource to be readable.
 ====
 
+[[color]]
+.--color
+[%collapsible]
+====
+Default: `auto` +
+When to format messages with ANSI color codes.
+Possible values:
+
+- `"never"`: Never format
+- `"auto"`: Format if the process' `stdin`, `stdout`, or `stderr` are connected to a console.
```

Comment:
`process'` is a britishism, maybe collapse to
> Format if `stdin`...

(same for EvaluatorSettings)

Reply:
TIL! (for my future reference; the britishism is that there no `s` after the `'`, right?)

Reply:
Yea `process's` (en-us) vs. `process'` (en-uk)

Reply:
TIL too; I also use a single apostrophe for this.